src/Microsoft.Health.Dicom.Api/Features/Conventions/ApiVersionsConvention.cs

```diff
@@ -30,14 +30,16 @@ internal class ApiVersionsConvention : IControllerConvention
         // this will result in null minor instead of 0 minor. There is no constructor on ApiVersion that allows this directly
         ApiVersion.Parse("1.0-prerelease"),
         ApiVersion.Parse("1"),
+        ApiVersion.Parse("2"),
     };
 
-    private static readonly IReadOnlyList<ApiVersion> UpcomingVersion = new List<ApiVersion>()
-    {
-        ApiVersion.Parse("2")
-    };
+    /// <summary>
+    /// Add upcoming API versions here so they can be used for private previews.
+    /// When upcomingVersion is ready for GA, move upcomingVersion to allSupportedVersion and remove from here.
+    /// </summary>
+    internal static IReadOnlyList<ApiVersion> UpcomingVersion = new List<ApiVersion>() { };
```

Comment:
I really don't like this being mutable, but I think the library we're using has been replaced in favor of another. So we can do this for now, with the idea that we'll be redoing our versioning logic